bears/vcs/git/GitCommitBear.py

```diff
@@ -66,6 +98,7 @@ def run(self, allow_empty_commit_message: bool = False, **kwargs):
             self.err('git:', repr(stderr))
             return
 
+        body = stdout.rstrip('\n')
```

Comment:
yes a separate commit makes it way easier to review. In fact you standardize and improve shortlog and body extraction :)